test/unittests/utils/test_filemanip.py

```diff
@@ -6,6 +6,62 @@
 from numpy.testing import assert_array_equal
 
 
+@pytest.mark.parametrize(
+    "dictionary, expected",
+    [
+        (
+            {
+                "TotalReadoutTime": 1,
+                "EstimatedTotalReadoutTime": "",
+                "PhaseEncodingSteps": "",
+                "PixelBandwidth": "",
+                "PhaseEncodingDirection": "j+",
+                "PhaseEncodingAxis": "",
+            },
+            [1, "j+"],
+        ),
+        (
+            {
+                "EstimatedTotalReadoutTime": 1,
+                "PhaseEncodingDirection": "j",
+            },
+            [1, "j"],
+        ),
+        (
+            {
+                "PhaseEncodingSteps": 1,
+                "PixelBandwidth": 0.5,
+                "PhaseEncodingDirection": "j-",
+            },
```

Comment:
This is a great start !
We are missing a few cases though:
- Missing PhaseEncodingDirection but Axis present
- checking error for zero pixel bandwidth
- checking all other errors when too much data is missing